packages/graph/hash_graph/bin/hash_graph/src/args.rs

```diff
@@ -1,17 +1,27 @@
-use clap::{Args as _, Command, Parser};
+use clap::Parser;
 use clap_complete::Shell;
 use graph::{logging::LoggingArgs, store::DatabaseConnectionInfo};
 use regex::Regex;
 
-/// Arguments passed to the program.
+/// Subcommand for the program.
+#[derive(Debug, clap::Subcommand)]
+pub enum Subcommand {
+    /// Run the Graph webserver.
+    Server(ServerArgs),
+    /// Run database migrations required by the Graph.
+    Migrate(MirgrateArgs),
+    /// Generate a completion script for the given shell and outputs it to stdout.
+    Completions(CompletionsArgs),
+}
+
 #[derive(Debug, Parser)]
 #[clap(version, author, about, long_about = None)]
-pub struct Args {
+pub struct ServerArgs {
```

Comment:
Sorry for being pedantic, but I'd prefer to have these in the subcommand files as well, so all subcommands are standalone.

Reply:
d374538afc71d74be00cefec0c7537844375cc2b seems nicer now, thanks :)

Reply:
Great, thanks! 🕺🏻